README.md

```diff
@@ -58,11 +58,14 @@ to always keep macOS on and available.
 
 **Usage:** `include_recipe 'macos::keep_awake'`
 
-| Attributes used                         | Default value           |
-|-----------------------------------------|-------------------------|
-| `node['macos']['remote_login_enabled']` | `true`                  |
-| `node['macos']['network_time_server']`  | `'time.windows.com'`    |
-| `node['macos']['time_zone']`            | `'America/Los_Angeles'` |
+| Attributes used                            | Default value           |
+|--------------------------------------------|-------------------------|
+| `node['macos']['remote_login_enabled']`    | `true`                  |
+| `node['macos']['vnc_screen_lock_enabled']` | `true`                  |
+| `node['macos']['network_time_server']`     | `'time.windows.com'`    |
+| `node['macos']['time_zone']`               | `'America/Los_Angeles'` |
```

Comment:
It's probably easier to specify the time zone in terms of a UTC offset so people don't have to guess at what magical string changes their time zone. That or list all the possible magic strings in the documentation.

Also, where does this get used?

Reply:
Nvm, I just realized this is a change to formatting to add a new attribute.

Reply:
we should probably map UTC offsets to the magic strings anyway - thanks for the suggestion!